docs/get-started/docker-compose.quickstart.yml

```diff
@@ -10,6 +10,8 @@ services:
       DATABASE_URL: postgres://postgres@database/postgres
       CONJUR_DATA_KEY:
     depends_on: [ database ]
+    ports:
+      - "3000:80"
```

Comment:
Can we use `8080` instead?

Reply:
I like 8080.  Let's get it in with that and then you're PR can include the link.